cypress/integration/Media/media.ts

```diff
@@ -0,0 +1,32 @@
+﻿/// <reference types="Cypress" />	
+context('Media', () => {
+
+    beforeEach(() => {
+        cy.umbracoLogin(Cypress.env('username'), Cypress.env('password'));
+        cy.umbracoSection("media");
+    });
+
+   
+        it('Create folder', () => {
+         //const folderName =
+            
+            cy.get(".dropdown-toggle").contains("Create").click({force: true});
+            cy.get('[role="menuitem"]').contains("Folder").click({force: true});
+            cy.get('[data-element="editor-name-field"]').type('Media Folder');
+            cy.umbracoButtonByLabelKey("buttons_save").click();
+        });
+        
+
+    it('Create file inside of folder', () => {
```

Comment:
We should also ensure that the media name does not exist before running the test (and save the media name in a const ;) )